src/ui/Forms/Translate/AutoTranslate.cs

```diff
@@ -792,10 +792,11 @@ public static string EvaluateDefaultTargetLanguageCode(string defaultSourceLangu
                 {
                     continue;
                 }
-                var iso639 = Iso639Dash2LanguageCode.GetTwoLetterCodeFromEnglishName(layoutName);
+
+                var iso639 = Iso639Dash2LanguageCode.GetTwoLetterCodeFromEnglishName(layoutName)?.ToLowerInvariant();
                 if (!string.IsNullOrEmpty(iso639) && !installedLanguages.Contains(iso639))
```

Comment:
The check was done without lowering `iso639.ToLowerInvariant()` but then when adding to `installedLanguages.Add(iso639.ToLowerInvariant());`  `.ToLowerInvariant()` was called.

This PR 🔗 fixes the inconsistency!